product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx

```diff
@@ -36,7 +36,7 @@ Column-level conflict resolution requires the table to have `REPLICA IDENTITY FU
 ## Enabling and disabling column-level conflict resolution
 
 !!! Note Permissions required
-Column-level conflict detection uses the `column_timestamps` type. This requires the user to have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
+Column-level conflict detection uses the `column_timestamps` type. This type requires you to have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned.
```

Comment:
@djw-m This negates our previous exchange...Not sure why you used second person here? If you mean a specific user (i.e., someone who's doing something specific) can you be more explicit?